sdk/python/kfp/dsl/_pipeline.py

```diff
@@ -104,6 +106,21 @@ def set_ttl_seconds_after_finished(self, seconds: int):
     self.ttl_seconds_after_finished = seconds
     return self
 
+  def set_pod_disruption_budget(self, min_available: str, selector: str=None):
+    """ PodDisruptionBudget holds the number of concurrent disruptions that you allow for pipeline Pods.
+	      Controller will automatically add the selector with workflow name, if selector is empty.
+
+    Args:
+        min_available (str):  An eviction is allowed if at least "minAvailable" pods selected by 
```

Comment:
I actually though it had to be a percentage but reading up it it seems like it could either be a percentage or a number. 
```minAvailable can be either an absolute number or a percentage. ``` [Link](https://kubernetes.io/docs/tasks/run-application/configure-pdb/#specifying-a-poddisruptionbudget). Will update the arg docs.